meson.build

```diff
@@ -0,0 +1,67 @@
+project(
+  'flatpak-builder',
+  'c',
+  license: 'LGPL-2.1-or-later',
+  meson_version: '>= 0.56.2',
+  version: '1.3.1',
+  default_options: 'c_std=gnu99',
+)
+
+cc = meson.get_compiler('c')
+
+add_project_arguments(
+  '-Werror=empty-body',
+  '-Werror=strict-prototypes',
+  '-Werror=missing-prototypes',
+  '-Werror=implicit-function-declaration',
+  '-Werror=format=2',
+  '-Werror=format-security',
+  '-Werror=format-nonliteral',
+  '-Werror=pointer-arith',
+  '-Werror=init-self',
+  '-Werror=missing-declarations',
+  '-Werror=return-type',
+  '-Werror=overflow',
+  '-Werror=int-conversion',
+  '-Werror=parentheses',
+  '-Werror=incompatible-pointer-types',
+  '-Werror=misleading-indentation',
+  '-Werror=missing-include-dirs',
+  # Necessary for non POSIX stuff in glibc like fallocate
+  '-D_GNU_SOURCE',
+  language: 'c',
+)
+
+dwarf_header_dir = get_option('dwarf_header_dir')
+
+inc_dirs = include_directories(
+  '.',
+# FIXME: handle $prefixes/libdwarf/ too or let it to the option? The later would be easier
+  # Dwarf header
+  dwarf_header_dir,
+)
+
+if not cc.has_header('dwarf.h', include_directories: inc_dirs)
```

Comment:
Can you do a `cc.find_library()` for this?

Reply:
After taking another look at this, I realized that all this code to find dwarf.h came from building `flatpak-builder-debugedit`. The Meson build doesn't do that - it hard depends on a system debugedit. This entire part of the build system can be removed! So can the `dwarf_header_dir` option.

Reply:
Is it really feasible already to assume system to have dwarf5 capable debugedit?

Reply:
It's reasonable to assume that anything equal or newer than Ubuntu 22.04 has that, which what the Meson build requires at minimum.

Reply:
Fair, looks like separate debugedit project has become common since when this feature was originally implemented.